site/_docs/history.md

```diff
@@ -57,6 +57,10 @@ other software versions as specified in gradle.properties.
 #### Bug-fixes, API changes and minor enhancements
 {: #fixes-1-37-0}
 
+* In the context of [CALCITE-6015] the visibility of the method
```

Comment:
@mihaibudiu I just noticed, this comment should better be placed a few lines up, right below  the 
```
#### Breaking Changes
{: #breaking-1-37-0}
```

Reply:
sorry about that, I have fixed this, and also squashed the commits